plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go

```diff
@@ -579,3 +582,28 @@ valuesInline:
 `)
 	th.AssertActualEqualsExpected(rm, "")
 }
+
+func TestTemplateCommand(t *testing.T) {
+	p := types.HelmChart{
+		Version:      "1.0.0",
+		Repo:         "https://helm.releases.hashicorp.com",
+		ApiVersions:  []string{"foo", "bar"},
+		Description:  "desc",
+		NameTemplate: "template",
+		SkipTests:    true,
+		IncludeCRDs:  true,
+		SkipHooks:    true,
+		ValuesFiles:  []string{"values1", "values2"},
+		Namespace:    "my-ns",
+	}
+	require.Equal(t, TemplateArgs(p),
```

Comment:
for whatever reason, if I change this to be a private function, and change this package from `main_test` to `main`, it complains that the function `templateArgs` is unknown. I'd really like to keep this unit test, so LMK you think making it public is not acceptable; any potential workarounds or suggestions are welcome.

Reply:
That's odd... I pulled your branch and made those changes, and I'm able to run the test against the private function. Maybe give it a fresh try?

Reply:
It was a failure in CI, not locally. I'll try changing it back to see if it mysteriously goes away.

Reply:
Took your suggestion offline to instead do a `(h HelmChart) AsHelmArgs []string` in `api/types`!